### ISSUE TITLE
avatar resizing questions

### ISSUE THREAD

**Issue by helenvarley** [upstage-org/limelight#304]
the shadow for the drag & drop avatar movement is very useful. can we also have this shadow for the resizing feature, so that you can see how big or small you are making your avatar?

& does the resizing happen as a jump, i.e. the audience see the avatar at one size and then immediately at the new size, or is it a gradual scaling, so that they see it shrinking or growing? (this would be cool - & maybe also remove the need for the shadow)

**Comment by rohrlach** [comment]
If we had a smooth scaling animation there'd be no need for the ghosting and no jump for audiences.

Which would be better?

**Comment by helenvarley** [comment]
i would say smooth scaling; there might be times when you want your avatar to suddenly be bigger or smaller, but that should be achieveable with a fast scale ... what do you think, @vickis ?

**Comment by thongteav** [comment]
We could have a smooth scaling on this issue. And a fast scale on another issue.

**Comment by rohrlach** [comment]
Sure - by 'which would be better' I'm really looking for 'which should be done first'

**Comment by vickis** [comment]
i think smooth scaling first
it seems to me as you suggest hen that once smooth is achieved fast should
follow

smooth as a priority for definite :)

**Comment by rohrlach** [comment]
This issue asks that:
- The resizing of avatars happen in real-time
- The resizing is visible to audience and other players
- The resizing appears as a 'shrink and grow' type effect

**Comment by charlie0605** [comment]
![resize-avatar](https://user-images.githubusercontent.com/25602185/38347308-c667aac0-38ee-11e8-9cdd-6a71c7d542e7.gif)
@rohrlach @helenvarley @vickis 
Does something like this look right?

**Comment by rohrlach** [comment]
Nice work @charlie0605 - That's exactly what I had in mind.

**Comment by helenvarley** [comment]
lovely! :)

**Comment by vickis** [comment]
Ha ha super! :D

**Comment by helenvarley** [comment]
cool :+1:
